File: common/src/main/java/org/sandboxpowered/silica/util/EmptyIOFilter.java
package org.sandboxpowered.silica.util;

import org.apache.commons.io.filefilter.IOFileFilter;

import java.io.File;

public class EmptyIOFilter implements IOFileFilter {
    @Override
    public boolean accept(File file) {
        return false;
    }

    @Override
    public boolean accept(File dir, String name) {
        return false;
    }
}

File: common/src/main/kotlin/org/sandboxpowered/silica/util/EmptyIOFilter.kt
package org.sandboxpowered.silica.util

import org.apache.commons.io.filefilter.IOFileFilter
import java.io.File

class EmptyIOFilter : IOFileFilter {
    override fun accept(file: File): Boolean = false

    override fun accept(dir: File, name: String): Boolean = false
}